packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -467,7 +467,7 @@ export class RpcClient implements RpcClientInterface {
       'hard_gas_limit_per_operation',
       'hard_gas_limit_per_block',
       'proof_of_work_threshold',
-      'tokens_per_roll',
```

Comment:
I don't think `tokens_per_roll` should be removed here, it is causing a breaking change as the type is changed from BigNumber to string.